hackathon_site/hardware/serializers.py

```diff
@@ -160,7 +160,7 @@ def get_team_code(obj: Order):
 
 class OrderChangeSerializer(OrderListSerializer):
     change_options = {
-        "Submitted": ["Cancelled", "Ready for Pickup"],
+        "Submitted": ["Cancelled", "Ready for Pickup"], # TODO: Unsure about whether I need to add the status "Pending" and "Rejected" to the list
```

Comment:
Yeah you should add Pending Status here, I believe it means from submitted status it can changed to  cancelled or ready for pickup. I'm thinking now that instead of pending we call it "Preparing" so that means admins are actively building the order.